### PR TITLE
Add table aws_auditmanager_control. Closes #402

### LINKED ISSUE #402: Add table aws_audit_manager_control
**References**
Add any related links that will help us understand the resource, including vendor documentation, related GitHub issues, and Go SDK documentation.

### PR DESCRIPTION
**Pull request by rajeshbal65**
# Integration test logs
<details>
  <summary>Logs</summary>
  
```
No env file present for the current environment:  staging
 Falling back to .env config
No env file present for the current environment:  staging
customEnv TURBOT_TEST_EXPECTED_TIMEOUT undefined

SETUP: tests/aws_auditmanager_control []

PRETEST: tests/aws_auditmanager_control

TEST: tests/aws_auditmanager_control
Running terraform
null_resource.named_test_resource: Creating...
null_resource.named_test_resource: Provisioning with 'local-exec'...
null_resource.named_test_resource (local-exec): Executing: ["/bin/sh" "-c" "      aws auditmanager create-control --name turbottest1871 --control-mapping-sources \"sourceName\"=turbottest1871,\"sourceType\"=\"AWS_Cloudtrail\",\"sourceSetUpOption\"=\"System_Controls_Mapping\",sourceKeyword=\"{keywordInputType=\"SELECT_FROM_LIST\",keywordValue=\"a4b_ApproveSkill\"}\" --tags name=turbottest1871 --profile default > /private/var/folders/11/p_45nj8d5xs3xfcyzd2ty6040000gn/T/tests/aws_auditmanager_control/terraform/test/control.json;\n"]
null_resource.named_test_resource: Creation complete after 3s [id=5204869495657362001]
data.local_file.control: Reading...
data.local_file.control: Read complete after 0s [id=cb3b7520e62cd47fa0df8e7229cab2f9268d987b]

Warning: Deprecated Resource

The null_data_source was historically used to construct intermediate values to
re-use elsewhere in configuration, the same can now be achieved using locals


Apply complete! Resources: 1 added, 0 changed, 0 destroyed.

Outputs:

account_id = "352685002396"
aws_partition = "aws"
aws_region = "us-east-1"
control_id = "5bf75266-95dd-405f-a411-dd4716ef687b"
resource_aka = "arn:aws:auditmanager:us-east-1:352685002396:control/5bf75266-95dd-405f-a411-dd4716ef687b"
resource_name = "turbottest1871"

Running SQL query: test-get-query.sql
[
  {
    "control_sources": "AWS CloudTrail",
    "id": "5bf75266-95dd-405f-a411-dd4716ef687b",
    "name": "turbottest1871",
    "type": "Custom"
  }
]
✔ PASSED

Running SQL query: test-list-query.sql
[
  {
    "control_sources": "AWS CloudTrail",
    "id": "5bf75266-95dd-405f-a411-dd4716ef687b",
    "name": "turbottest1871",
    "type": "Custom"
  }
]
✔ PASSED

Running SQL query: test-notfound-query.sql
null
✔ PASSED

Running SQL query: test-turbot-query.sql
[
  {
    "akas": [
      "arn:aws:auditmanager:us-east-1:352685002396:control/5bf75266-95dd-405f-a411-dd4716ef687b"
    ],
    "tags": {
      "name": "turbottest1871"
    },
    "title": "turbottest1871"
  }
]
✔ PASSED

POSTTEST: tests/aws_auditmanager_control
Running terraform
null_resource.named_test_resource: Creating...
null_resource.named_test_resource: Provisioning with 'local-exec'...
null_resource.named_test_resource (local-exec): Executing: ["/bin/sh" "-c" "aws auditmanager delete-control --control-id 5bf75266-95dd-405f-a411-dd4716ef687b --profile default"]
null_resource.named_test_resource: Creation complete after 2s [id=629936962759439479]

Warning: Deprecated Resource

The null_data_source was historically used to construct intermediate values to
re-use elsewhere in configuration, the same can now be achieved using locals


Apply complete! Resources: 1 added, 0 changed, 0 destroyed.

TEARDOWN: tests/aws_auditmanager_control


SUMMARY:

1/1 passed.
```
</details>

# Example query results
<details>
  <summary>Results</summary>
  
```sql
select
  name,
  id,
  description,
  type
from
  aws_auditmanager_control;
```
```
+--------------------------------------------------+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------+
| name                                             | id                                   | description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | type     |
+--------------------------------------------------+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------+
| Article 82 Right to compensation and liability.5 | b312fb1f-145b-4bf3-94be-d857a161c500 | 5. Where a controller or processor has, in accordance with paragraph 4, paid full compensation for the damage suffered, that controller or processor shall be entitled to claim back from the other controllers or processors involved in the same processing that part of the compensation corresponding to their part of responsibility for the damage, in accordance with the conditions set out in paragraph 2.                                                                                                                                                                                                                                                                                                | Standard |
| Article 99 Entry into force and application.2    | b54bb9b9-7461-496f-8475-b741509b4737 | 2. It shall apply from â€¦ [two years from the date of entry into force of this Regulation].                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | Standard |
| Article 52 Independence.5                        | ed08d230-611e-46ee-8e90-3811f798534d | 5. Each Member State shall ensure that each supervisory authority chooses and has its own staff which shall be subject to the exclusive direction of the member or members of the supervisory authority concerned.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | Standard |
| Article 90 Obligations of secrecy.1              | 4450c4df-f3ea-42e1-aad0-5b386eaf2ce8 | 1. Member States may adopt specific rules to set out the powers of the supervisory authorities laid down in points (e) and (f) of Article 58(1) in relation to controllers or processors that are subject, under Union or Member State law or rules established by national competent bodies, to an obligation of professional secrecy or other equivalent obligations of secrecy where this is necessary and proportionate to reconcile the right of the protection of personal data with the obligation of secrecy. Those rules shall apply only with regard to personal data which the controller or processor has received as a result ofor has obtained in an activity covered by that obligation of secrecy. | Standard |
| Article 82 Right to compensation and liability.3 | d46b06c5-89d2-4613-86c7-7fcdafec83fc | 3. A controller or processor shall be exempt from liability under paragraph 2 if it proves that it is not in any way responsible for the event giving rise to the damage.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | Standard |
+--------------------------------------------------+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------+
```


### List custom audit manager controls

```sql
select
  name,
  id,
  type
from
  aws_auditmanager_control
where
  type = 'Custom';
```
```
+-----------------+--------------------------------------+--------+
| name            | id                                   | type   |
+-----------------+--------------------------------------+--------+
| turbottest63785 | 943aa340-5edc-4929-8851-de592c510b86 | Custom |
+-----------------+--------------------------------------+--------+
```
</details>

### CODE REVIEW

**Comment by rajeshbal65** [comment]
Integration test To be added.

**Review by Subhajit97 — APPROVED** [review]
LGTM

**Review by rajmohanty17 — APPROVED** [review]
LGTM

File: docs/tables/aws_audit_manager_control.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  name,
+  id,
+  description,
+  type
+from
+  aws_audit_manager_control;
+```
+
+
+### List custom audit manager controls
```
Comment (rajmohanty17) [inline docs/tables/aws_audit_manager_control.md:20]: @rajeshbal65 @Subhajit97 is it going to give me list of `Custom frameworks`
image

**Comment by cbruno10** [comment]
In TF, even though Audit Manager resources are not available, it looks like they're using the format `aws_auditmanager_*` (https://github.com/hashicorp/terraform-provider-aws/blob/ceb4c185a26c1635b6f3d26f3adce1be301a3667/.hashibot.hcl#L141), so we should follow the same format. 

Can you please rename the table and files to use this naming structure instead?

File: docs/tables/aws_audit_manager_control.md (hunk 1 above)
Comment (Subhajit97) [reply docs/tables/aws_audit_manager_control.md:20]: @rajmohanty17 We have a separate table (aws_auditmanager_framework) to list the `assessment framework`.
This table will list auditmanager controls (i.e. both Custom and Standard).

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**